src/ActionList/Item.tsx

```diff
@@ -34,6 +34,7 @@ const InactiveIndicator: React.FC<{
         padding: 0,
         font: 'inherit',
         cursor: 'pointer',
+        display: 'flex',
```

Comment:
Due to removing the wrapper in the tooltip, I needed to add this to align the inactive visual in center. Let me know if you prefer a different way to do it @mperrotti